dashboard/app/coverage.go

```diff
@@ -71,6 +71,55 @@ func handleSubsystemsCoverageHeatmap(c context.Context, w http.ResponseWriter, r
 	return handleHeatmap(c, w, r, cover.DoSubsystemsHeatMapStyleBodyJS)
 }
 
+type covPageParam int
+
+const (
+	PeriodType covPageParam = iota
+	PeriodCount
+	DateTo
+	MinCoverLinesDrop
+	OrderByCoverDrop
+	UniqueOnly
+	ManagerName
+	SubsystemName
+	CommitHash
+	FilePath
+)
+
+var covPageParams = map[covPageParam]string{
+	PeriodType:        "period",
+	PeriodCount:       "period_count",
+	DateTo:            "dateto",
+	MinCoverLinesDrop: "min-cover-lines-drop",
+	OrderByCoverDrop:  "order-by-cover-lines-drop",
+	UniqueOnly:        "unique-only",
+	ManagerName:       "manager",
+	SubsystemName:     "subsystem",
+	CommitHash:        "commit",
+	FilePath:          "filepath",
+}
+
+func coveragePageLink(ns, periodType, dateTo string, minDrop, periodCount int, orderByCoverDrop bool) string {
+	if periodType == "" {
+		periodType = coveragedb.MonthPeriod
+	}
+	res := "/" + ns + "/coverage?" + covPageParams[PeriodType] + "=" + periodType
+
+	if periodCount != 0 {
+		res += "&" + covPageParams[PeriodCount] + "=" + strconv.Itoa(periodCount)
+	}
+	if dateTo != "" {
```

Comment:
Nit: can we avoid repeating the code that constructs URL parameters?
Collecting the necessary parameters into a dict and then joining it would already look cleaner, and maybe there's even a library function to do that.

Reply:
Let's use urlutil. PTAL.

Reply:
LG